src/views/projectExplorer/index.ts

```diff
@@ -696,11 +696,12 @@ export default class ProjectExplorer implements TreeDataProvider<ProjectExplorer
                 }
 
                 const variable = await window.showQuickPick(variableItems, {
-                  placeHolder: l10n.t('Select a variable')
+                  placeHolder: l10n.t('Select  a variable')
```

Comment:
you probably don't want this extra space

Reply:
It's outdated, it's been removed yesterday 😉